extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -248,6 +247,27 @@ export async function getCompletionItemsFromSpecs(
 					command.documentation
 				));
 				labels.add(commandTextLabel);
+			} else {
+				// Potential builitin that should override the spec's description
+				let score = typeof command.label === 'object' ? (command.label.detail !== undefined ? 2 : 0) : 0;
+				score += typeof command.label === 'object' ? (command.label.description !== undefined ? 1 : 0) : 0;
```

Comment:
`command.documentation` is also a big signal. We should just add in all the properties here, including `command.detail` and `command.documentation`. Also `command.documentation` as a markdown string should beat a string.